README.md

```diff
@@ -11,7 +11,7 @@ To build the Anthology website, you will need:
 
 + **Python 3.8** or higher
 + Python packages listed in `bin/requirements.txt`; to install, run `pip -r bin/requirements.txt`
-+ [**Hugo 0.58.3**](https://gohugo.io) or higher (can be [downloaded directly from
++ [**Hugo 0.108.0**](https://gohugo.io) or higher (can be [downloaded directly from
```

Comment:
If you want to set this as the new minimal version, the version check in the Makefile should be adjusted as well.

Reply:
I'll revert it.